djangoplicity/actions/models.py

```diff
@@ -153,7 +153,7 @@ class ActionParameter( models.Model ):
     """
     Parameter for an action
     """
-    action = models.ForeignKey( Action )
+    action = models.ForeignKey( Action, on_delete=models.CASCADE )
```

Comment:
I think this should not be included in this PR because it is related to the Django update rather than the Python update